client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/InteractedContactAccordion.tsx

```diff
@@ -36,6 +36,8 @@ const InteractedContactAccordion = (props: Props) => {
         shouldDisable,
     } = props;
 
+    const isStatusCompleted = interactedContact.contactStatus === 5
```

Comment:
here as well

Reply:
I solved this case in the schema so I removed it...